src/runtime/runtime.cs

```diff
@@ -118,16 +119,16 @@ public class Runtime
             { "Linux", OperatingSystemType.Linux },
         };
 
-        /// <summary>
-        /// Gets the operating system as reported by python's platform.system().
-        /// </summary>
-        public static OperatingSystemType OperatingSystem { get; private set; }
+        [Obsolete]
+        public static string OperatingSystemName => Enum.GetName(typeof(OperatingSystemType), OperatingSystem);
```

Comment:
NIT: Is there a reason why you can't just do NIT: is there a reason why can't you just do `OperatingSystem.ToString()`?

Reply:
Just incompetence, I simplified it.